draft-packages/menu/KaizenDraft/Menu/MenuDropdown.tsx

```diff
@@ -14,20 +14,27 @@ type MenuDropdownProps = {
   } | null
   align?: "left" | "right"
   width?: "default" | "contain"
+  autoHideOnClick?: boolean
 }
 
 class MenuDropdown extends React.Component<MenuDropdownProps> {
   static displayName = "MenuDropdown"
   menu = React.createRef<HTMLDivElement>()
 
   componentDidMount() {
-    document.addEventListener("click", this.handleDocumentClick, false)
+    const { autoHideOnClick } = this.props
+    if (autoHideOnClick) {
+      document.addEventListener("click", this.handleDocumentClick, false)
+    }
```

Comment:
So it looks like when you remove the event listener, it also removes the logic that hides the dropdown when clicking outside of the dropdown. I presume we don't want this.